.github/contributing/new-resource.md

```diff
@@ -1,10 +1,12 @@
-# Adding a new resource type to driftctl
-![Diagram](media/resource.png)
+# Add new resources
 
-## 1 Defining the resource
+![Diagram](../assets/resource.png)
```

Comment:
> This is broken when viewing the markdown file directly. Is it possible to make it work for both markdown and generated doc website ?

My bad I forgot to change the link. It will be viewable on both md file and website, don't worry.

> Btw why did you moved doc/contributing/media/resource.png to 

Since this is not meant to be on the website as per our call this morning, and for docusaurus to not copy for versioned docs purposes, I needed it to be outside the `docs` folder.

Reply:
I think `.github` is not the right place, I close this thread maybe we could continue this discussion in my other remark there : https://github.com/cloudskiff/driftctl/pull/245#discussion_r574504420